src/Reflector/Reflector.php

```diff
@@ -52,4 +53,11 @@ public function reflectConstant(string $identifierName): ReflectionConstant;
      * @return list<ReflectionConstant>
      */
     public function reflectAllConstants(): iterable;
+
+    /**
+     * Get all attributes available in the scope specified by the SourceLocator.
+     *
+     * @return List<ReflectionAttribute>
```

Comment:
Hmm, not sure if case matters to Psalm, but I think it should be `list<ReflectionAttribute>` for consistency either way